sale_timesheet_rounded/tests/test_rounded.py

```diff
@@ -162,7 +165,7 @@ def test_analytic_line_read_override(self):
         unit_amount_ret = line.with_context(timesheet_rounding=True).read(fields, load)[
             0
         ]["unit_amount"]
-        self.assertEqual(unit_amount_ret, 1)
+        self.assertEqual(unit_amount_ret, 2)
```

Comment:
why did the return value change to 2 instead of 1? The same input should also end up in the same result. (if there was no error before)

Reply:
Did you check the changed unit test case that everything works as expected?

Yes, I checked it, I checked the way how this test works and my opinion is that we need to change value from 1 to 2, in 14.0 version there is no way to get 1 value in this case.

Reply:
The header says:
     # * not linked to project -> no impact

The line is created with project_id = False, and than it should not have any rounding. Therefore i would assume that the result should be 1.